source/help/apps/desktop-guide.rst

```diff
@@ -4,32 +4,32 @@ Using Mattermost Desktop App
 
 Mattermost desktop applications are available for Windows, Mac and Linux operating systems. 
 
-You can `download the apps directly from our download page <https://about.mattermost.com/downloads/>`_ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`_ for help during set up.
+You can `download the apps directly from our download page <https://about.mattermost.com/downloads/>`_ and visit our `installation guides <https://docs.mattermost.com/install/desktop.html>`_ for help during set up and for troubleshooting tips.
 
 To view the latest updates, please see our `changelog <https://docs.mattermost.com/help/apps/desktop-changelog.html>`_.
 
-Below are a few tips to get you started and configuring your experience on the desktop app:
+Below are a few tips to get you started and to configure your experience on the desktop app:
 
  - `Team Management <https://docs.mattermost.com/help/apps/desktop-guide.html#id1>`_
  - `App Options <https://docs.mattermost.com/help/apps/desktop-guide.html#id2>`_
  - `Menu Bar <https://docs.mattermost.com/help/apps/desktop-guide.html#id3>`_
- - `Troubleshooting <https://docs.mattermost.com/help/apps/desktop-guide.html#id4>`_
 
 Team Management
 ---------------------------------------------------------------------
 
 You can connect to multiple Mattermost servers from a single interface on the desktop app.
 
-Team Management appears in the Settings page, available from the `File` menu under `Settings`.
+The Team Management section allows you to add, edit and remove servers and teams.
 
 Adding servers and teams
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 To add a new server on the Mattermost instance, 
 
-1. Click **Add new team** next to the right of Team Management section.
-2. Enter **Name** and a valid **URL**, which begins with either ``http://`` or ``https://``.
-3. Click **Add**.
```

Comment:
The thing with the current desktop release is that the top menu bar might be hidden for users, depending on the "Hide Menu Bar" option.

So I think it's better to give both in this case.

This "Hide Menu Bar" option is planned to be removed in the next desktop release as well -- once done, I can update this step to remove the "CTRL/CMD+COMMA" part